todo_app/view_model_test.py

```diff
@@ -0,0 +1,59 @@
+
+from todo_app.data.item import Item
+from todo_app.data.view_model import ViewModel
+
+def test_todo_items_property_only_shows_todo_items_and_nothing_else():
+    # arrange
+    items = [
+        Item(1, "Test Item 1", "To do"),
+        Item(2, "Test Item 2", "Doing"),
+        Item(3, "Test Item 3", "Done")
+
+    ]
+    view_model = ViewModel(items)
+
+    #act
+    returned_items = view_model.todo_items
+
+    # assert
+    assert len(returned_items) == 1
+    returned_single_item = returned_items [0]
+    assert returned_single_item.status == "To do"
+
+
+def test_doing_items_property_only_shows_doing_items_and_nothing_else():
+    # arrange
+    items = [
+        Item(1, "Test Item 1", "To do"),
+        Item(2, "Test Item 2", "Doing"),
+        Item(3, "Test Item 3", "Done")
+
+    ]
+    view_model = ViewModel(items)
+
+    #act
+    returned_items = view_model.doing_items
+
+    # assert
+    assert len(returned_items) == 1
+    returned_single_item = returned_items [0]
+    assert returned_single_item.status == "Doing"
+
+
+def test_done_items_property_only_shows_done_items_and_nothing_else():
```

Comment:
Nice - these tests are clean and well targeted 😄 
One thing we could potentially do to just save some repetition is to extract some of the setup to a fixture, e.g. something like:
```python
@pytest.fixture
def items():
    return [
        Item(1, "Test Item 1", "To do"),
        Item(2, "Test Item 2", "Doing"),
        Item(3, "Test Item 3", "Done")
    ]

def test_todo_items_property_only_shows_todo_items_and_nothing_else(items):
    # arrange
    view_model = ViewModel(items)
    ....
```

One disadvantage of this is that changing the list of items might affect lots of different tests (e.g. it's hard to tell whether adding a second "To do" item will break other tests) so it's not a clear win